app/presenters/finder_presenter.rb

```diff
@@ -4,6 +4,7 @@ class FinderPresenter
 
   delegate :beta_message,
            :document_noun,
+           :context,
```

Comment:
Is `context` the best name for the this? I think it might be too broad. How about something like `title_context`?